src/devices/video/gb_lcd.h

```diff
@@ -262,6 +256,8 @@ class sgb_ppu_device : public dmg_ppu_device
 	virtual void update_sprites() override;
 	virtual void update_scanline(uint32_t cycles_to_go) override;
 	void refresh_border();
+	
+	void sgb_vram_memcpy(uint8_t* dst, uint8_t start, size_t num_tiles);
```

Comment:
The `*` is unary and binds to the thing on it right.  Consider this:
```
char* a, b;
```
This is misleading - the `*` doesn't bind to `char`, it binds to `a`.  It's equivalent to:
```
char *a;
char b;
```

This is less misleading and closer to what the parser does, because the `*` is next to what it binds to:
```
char *a, b;
```